bin/node/src/chain_spec.rs

```diff
@@ -201,38 +201,26 @@ fn system_properties(token_symbol: String) -> serde_json::map::Map<String, Value
     .collect()
 }
 
-/// Generate chain spec.
-pub fn config(
-    chain_params: ChainParams,
-    authorities: Vec<AuthorityKeys>,
-) -> Result<ChainSpec, String> {
-    generate_chain_spec_config(chain_params, authorities, None, None)
-}
-
 /// Generate chain spec for local runs.
-/// Stash and controller accounts are generated for the specified authorities.
+/// Controller accounts are generated for the specified authorities.
 pub fn local_config(
```

Comment:
now, when `config` is gone, I would rename `local_config` to just `config`, since as I understand, this is used in _all_ environments, not only local setting, right?

Reply:
Good point